src/index.ts

```diff
@@ -32,6 +32,7 @@ import { stringify } from './stringify.js'
 import { TomlDate } from './date.js'
 import { TomlError } from './error.js'
 
-export type { TomlPrimitive } from './util.js'
+export type { TomlValue, TomlTable } from './util.js'
+export type { TomlValue as TomlPrimitive } from './util.js' // deprecated, to be removed in v2.x
```

Comment:
Should be marked using [`@deprecated`](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#deprecated) so editors and linters can pick it up

Reply:
I did, but that doesn't work on my end